Dev david issue 460

davidszkiba opened the pull request:
Closes #460

eynimeni reviewed (commented):
why string and not int?

davidszkiba commented:
> why string and not int?

Internally, the status class is used to indicate the stop reason for a quiz attempt. And this was based on strings from the beginning. Since the DB column for the status needs an INT value, I added a function that converts each status to an INT value.
 I updated the switch statement  to use the strings from the constants instead of string literals.
